.travis.yml

```diff
@@ -45,7 +45,7 @@ install:
 before_script:
     - if [ "$CHECK_DOC" = 1 -a "$TRAVIS_EVENT_TYPE" = "pull_request" ]; then contrib/devtools/commit-script-check.sh $TRAVIS_COMMIT_RANGE; fi
     - if [ "$CHECK_DOC" = 1 ]; then contrib/devtools/check-doc.py; fi
-    - unset CC; unset CXX
```

Comment:
Is this removal intended? How is it related to this PR?

Reply:
Sigh, rebase fail